apps/web/components/blocks/Image/Image.vue

```diff
@@ -1,11 +1,17 @@
 <template>
-  <div class="relative w-full" :class="['flex justify-center']" :style="[wrapperStyle]">
+  <div
+    class="relative w-full"
+    :class="['flex justify-center']"
```

Comment:
```suggestion
    class="relative w-full flex justify-center"
```
I know it's not from your change, but we only need one `class`, right?